var/spack/repos/builtin/packages/neurodamus-model/package.py

```diff
@@ -99,69 +79,31 @@ def build(self, spec, prefix):
-        if '+coreneuron' in spec:
-            include_flag += ' -I%s' % (spec['coreneuron'].prefix.include)
-            which('nrnivmodl-core')(
-                '-i', include_flag, '-l', link_flag, '-n', self.mech_name,
-                '-v', str(spec.version), '-c', '_core_mechs')
-            output_dir = spec.architecture.target
-            mechlib = find_libraries("libcorenrnmech*", output_dir)
-            assert len(mechlib), "Error creating corenrnmech lib"
-
-            # Link neuron special with this mechs lib
-            link_flag += ' ' + mechlib.ld_flags + self._get_lib_flags('coreneuron')
-            include_flag += ' -DENABLE_CORENEURON'  # only now, otherwise some mods assume neuron
-
-        with profiling_wrapper_on():
-            which('nrnivmodl')('-incflags', include_flag, '-loadflags', link_flag, 'm')
-        special = os.path.join(os.path.basename(self.neuron_archdir), 'special')
-        assert os.path.isfile(special)
-
-    def _get_lib_flags(self, lib_name):
-        """ Helper method to get linking flags similar to spack build, for solid deployments:
-              1. static libs passed via full path
-              2. shared libs passed with -L -l and RPATH flags
-            Attention: This func doesnt handle recursive deps of static libs.
-        """
-        spec = self.spec[lib_name]
-        if spec.satisfies('+shared'):  # Prefer shared if both exist
-            return " %s %s" % (spec.libs.rpath_flags, spec.libs.ld_flags)
-        return ' ' + spec.libs.joined()
+        # Override mech_name in order to generate a library with a different name
+        self.mech_name += '_damus'
+        self._build_mods('mod', link_flag, include_flag, 'mod_core')
 
+    # -
     def install(self, spec, prefix):
         """ Install:
               bin/ <- special and special-core
               lib/ <- hoc, mod and lib*mech*.so
               share/ <- neuron & coreneuron mod.c's (modc and modc_core)
               python/ If neurodamus-core comes with python, create links
         """
-        mkdirp(prefix.bin)
-        mkdirp(prefix.lib)
-        mkdirp(prefix.share.modc)
-        shutil.move('_merged_hoc', prefix.lib.hoc)
-        shutil.move('_merged_mod', prefix.lib.mod)
-
-        arch = os.path.basename(self.neuron_archdir)
-        shutil.move(join_path(arch, 'special'), prefix.bin)
-
-        # Copy c mods (for neuron)
-        for cmod in find(arch, "*.c", recursive=False):
-            shutil.move(cmod, prefix.share.modc)
-
-        # Handle non-binary special
-        if os.path.exists(arch + "/.libs/libnrnmech.so"):
-            shutil.move(arch + "/.libs/libnrnmech.so", prefix.lib)
-            sed = which('sed')
-            sed('-i', 's#-dll .*#-dll %s#' % prefix.lib.join('libnrnmech.so'),
-                prefix.bin.special)
+        # base dest dirs already created by model install
+        arch = arch = spec.architecture.target
+        dst_libname = 'libnrnmech_damus.so'
```

Comment:
naming question again :D  why  not full `neurodamus` instead of `damus` ?  I don't think there is any meaning to `damus` :)

Reply:
it's a shorthand. Is it too bad? Sure we can change

Reply:
Does this actually need to be renamed ?

Reply:
This is the library internal to neurodamus (which includes aux mods). Has to be renamed to avoid overwriting the original libnrnmech which is what you need, right?

Reply:
if there is an libnrnmech in the same dir, then yes, renaming doesn't make sense.
Otherwise, I'm not sure if it couldn't have the same name.

Reply:
I guess there's some confusion here. The recipe is renaming the new one.
The env var `BGLIBPY_MOD_LIBRARY_PATH` will point to the original `libnrnmech.so` which doesnt contain the the util mods

Reply:
I updated the PR description to be clearer in this regard.